vivisect/base.py

```diff
@@ -22,6 +23,8 @@
 from vivisect.exc import *
 from vivisect.const import *
 
+logger = logging.getLogger(__name__)
```

Comment:
So are we standardizing on logger instead of vprint now? What about the the cases where we'd like to print things out to the UI?

Reply:
logger is the path forward, particularly for library things.  @invisig0th has made a hard line that library calls should not print out anything, but logging things is different.  we haven't made a "all Vivisect logging pull request" yet, kinda like we haven't done "flake8" pr... but it's in process.